### ISSUE TITLE
Error in plugin "gulp-postcss" -- [object Object] is not a PostCSS plugin

### ISSUE THREAD

**Issue by jesperalmstrom** [TryGhost/London#74]
Hi!
Following your guide, I did run `yarn install` successfully. When running `yarn dev` i get the following error 
```
yarn run v1.13.0
$ gulp
[18:43:03] Using gulpfile ~/dev/London/gulpfile.js
[18:43:03] Starting 'default'...
[18:43:03] Starting 'css'...
[18:43:03] 'css' errored after 21 ms
[18:43:03] Error in plugin "gulp-postcss"
Message:
    [object Object] is not a PostCSS plugin
Details:
    fileName: /Users/jespera/dev/London/assets/css/screen.css

Stack:
Error: [object Object] is not a PostCSS plugin
    at Processor.normalize (/Users/jespera/dev/London/node_modules/postcss/lib/processor.js:168:15)
    at new Processor (/Users/jespera/dev/London/node_modules/postcss/lib/processor.js:52:25)
    at postcss (/Users/jespera/dev/London/node_modules/postcss/lib/postcss.js:55:10)
    at /Users/jespera/dev/London/node_modules/gulp-postcss/index.js:50:16
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
[18:43:03] 'default' errored after 23 ms
error Command failed with exit code 1.
```
Is there something missing? Have you seen this issue before?

**Comment by roketstorm** [comment]
Got the same issue

**Comment by ServantWolf** [comment]
I was able to work around it by running:

```
npm install
npm install autoprefixer@9.8.6
```

**Comment by jesperalmstrom** [comment]
> I was able to work around it by running:
> 
> 
> 
> ```
> 
> npm install
> 
> npm install autoprefixer@9.8.6
> 
> ```

Thanks. Works fine!

**Comment by minimaluminium** [comment]
PostCSS now should be installed as a peer dependency in `gulp-postcss` [v9](https://github.com/postcss/gulp-postcss#changelog). Fixed in 0e7fe41.
